config/register-api.yml

```diff
@@ -285,7 +285,7 @@ components:
           enum:
           - male
           - female
-          - intersex
+          - other
```

Comment:
Is it all aligned on Register?

Reply:
Not yet, they'll need to add other to the mapping as they have intersex https://github.com/DFE-Digital/register-trainee-teachers/blob/main/app/lib/apply_api/code_sets/genders.rb

Reply:
Merged on Register https://github.com/DFE-Digital/register-trainee-teachers/pull/2807